readme.md

```diff
@@ -17,31 +17,32 @@ $ npm install --save jsftp-mkdirp
 ## Usage
 
 ```js
-var JSFtp = require('jsftp');
+let JSFtp = require('jsftp');
 
 // decorate `JSFtp` with a new method `mkdirp`
 JSFtp = require('jsftp-mkdirp')(JSFtp);
 
-var path = 'public_html/deploy/foo/bar';
+const path = 'public_html/deploy/foo/bar';
 
-ftp.mkdirp(path, function (err) {
+ftp.mkdirp(path).then(() => {
 	console.log('Created path:', path);
 });
 ```
 
 
 ## API
 
-### JSFtp.mkdirp(path, callback)
+### JSFtp.mkdirp(path)
+
+Returns a Promise.
```

Comment:
@kevva `promise` looks better and what I've been using, but `Promise` is more correct as it's a name.